packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -70,24 +67,24 @@ class DiffPaneController {
   /// informational item.
   bool get hasSnapshots => snapshots.value.length > 1;
 
-  final classSorting = ColumnSorting();
-
-  final classDiffSorting = ColumnSorting();
-
-  final pathSorting = ColumnSorting();
+  // This value should never be reset. It is incremented for every snapshot that
+  // is taken, and is used to assign a unique id to each [SnapshotListItem].
+  int _snapshotId = 0;
```

Comment:
do you need to merge with master? this doesn't look like it should be part of your PR since it was from the Table rewrite PR